src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/LinearCopyRatioCollection.java

```diff
@@ -64,13 +64,13 @@ public DenoisedLocatableCopyRatioCollection(final SampleLocatableMetadata metada
     }
 
     /**
-     * Generates an instance of {@link DataLine} from {@link DenoisedLocatableCopyRatio}
+     * Generates an instance of {@link DataLine} from {@link LinearCopyRatio}
      */
-    private static BiConsumer<DenoisedLocatableCopyRatio, DataLine> getDenoisedLocatableCopyRatioRecordToDataLineEncoder() {
-        return (denoisedLocatableCopyRatioCopyRatio, dataLine) -> dataLine.append(denoisedLocatableCopyRatioCopyRatio.getContig())
-                .append(denoisedLocatableCopyRatioCopyRatio.getStart())
-                .append(denoisedLocatableCopyRatioCopyRatio.getEnd())
-                .append(denoisedLocatableCopyRatioCopyRatio.getDenoisedCopyRatio().getDenoisedCopyRatio());
+    private static BiConsumer<LinearCopyRatio, DataLine> getDenoisedLocatableCopyRatioRecordToDataLineEncoder() {
```

Comment:
Should this function be renamed to `getLinearCopyRatioRecordToDataLineEncoder`?

Reply:
Yes, good catch